client/js/content.js

```diff
@@ -0,0 +1,26 @@
+var images = document.getElementsByTagName('img');
+// for (var i = 0, l = images.length; i < l; i++) {
+//   images[i].src = 'http://placekitten.com/' + images[i].width + '/' + images[i].height;
+// }
+
+var textbox = document.getElementsByClassName("episode")
+for (var i = 0, l = textbox.length; i<l; i++){
+    console.log(textbox[i])
+    var imdbRatings = document.createElement('div')
+    imdbRatings.innerHTML = "IMDB Ratings : 5.0"
+    imdbRatings.style.width = '100%';
+    imdbRatings.style.height = '100%';
+    imdbRatings.style.border = '1px solid red';
+    imdbRatings.style.float = 'left';
+
+    var rottenTomatoRatings = document.createElement('div')
```

Comment:
Try using let. The var scope is global and it's not block scoped. This may lead to random bugs in future which are hard o catch